docs/source/models.rst

```diff
@@ -1939,6 +1939,249 @@ For a ``GroupedPmdarima`` model, an example configuration for the ``pyfunc`` ``p
+and batch inference by coercing the data structures that are required for ``transformers`` inference pipelines
+to formats that are compatible with json serialization and casting to Pandas DataFrames.
+
+.. attention::
+    Not all ``transformers`` pipeline types are supported. See the table below for the list of currently supported Pipeline
+    types that can be loaded as ``pyfunc``.
+
+    In the current version, text-based large language
+    models are supported for use with ``pyfunc``, while computer vision, audio, multi-modal, timeseries,
+    reinforcement learning, and graph models are only supported for native type loading via :py:func:`mlflow.transformers.load_model()`
+
+    Future releases of MLflow will introduce ``pyfunc`` support for these additional types.
+
+The table below shows the mapping of ``transformers`` pipeline types to the :ref:`python_function (pyfunc) model flavor <pyfunc-model-flavor>`
+data type inputs and outputs.
+
+.. important::
+    The inputs and outputs of the ``pyfunc`` implementation of these pipelines *are not guaranteed to match* the input types and output types that would
+    return from a native use of a given pipeline type. If your use case requires access to scores, top_k results, or other additional references within
+    the output from a pipeline inference call, please use the native implementation by loading via :py:func:`mlflow.transformers.load_model()` to
+    receive the full output.
+
+    Similarly, if your use case requires the use of raw tensor outputs or processing of outputs through an external ``processor`` module, load the
+    model components directly as a ``dict`` by calling :py:func:`mlflow.transformers.load_model()` and specify the ``return_type`` argument as 'components'.
+
+Supported transformers Pipeline types for Pyfunc
+""""""""""""""""""""""""""""""""""""""""""""""""
+
+================================= ================================= =================
+Pipeline Type                     Input Type                        Output Type
+================================= ================================= =================
+Conversational                    str or List[str]                  str or List[str]
+Summarization                     str or List[str]                  str or List[str]
+Text Classification               str or List[str]                  str or List[str]
+Text Generation                   str or List[str]                  str or List[str]
+Text2Text Generation              str or List[str]                  str or List[str]
+Token Classification              str or List[str]                  str or List[str]
+Translation                       str or List[str]                  str or List[str]
+ZeroShot Classification*          Dict[str, Union[List[str]|str]]*  str or List[str]
+Table Question Answering**        Dict[str, Union[List[str]|str]]** str or List[str]
```

Comment:
```suggestion
ZeroShot Classification*          Dict[str, List[str] | str]*  str or List[str]
Table Question Answering**        Dict[str, List[str] | str]** str or List[str]
```

Reply:
For ZeroShot Classification, is `{"<key1>": "val1", <key1>: ["val2"]}` a valid input? `Dict[str, List[str] | str]` implies it is.

Reply:
I like this better. Easier to read :)